src/worker/queue.ts

```diff
@@ -72,15 +72,16 @@ async function startQueueKafka(
     processEventBatch: (event: PluginEvent[]) => Promise<(PluginEvent | null)[]>
 ): Promise<Queue> {
     const kafkaQueue = new KafkaQueue(server, processEventBatch, async (event: PluginEvent) => {
-        const { distinct_id, ip, site_url, team_id, now, sent_at } = event
-        await server.eventsProcessor.process_event_ee(
+        const { distinct_id, ip, site_url, team_id, now, sent_at, uuid } = event
+        await server.eventsProcessor.processEventEE(
             distinct_id,
             ip,
             site_url,
             event,
             team_id,
             DateTime.fromISO(now),
-            sent_at ? DateTime.fromISO(sent_at) : null
+            sent_at ? DateTime.fromISO(sent_at) : null,
+            uuid!
```

Comment:
This `!` feels a bit unsafe, though I guess that's the simplest way to assert that it's always there, which it should be for EE. Not sure if anything to do (`if (!uuid) { console.error && return }`?), just thinking out loud again :).

Reply:
That's fair, will add a check